inc/Database/Exporter.php

```diff
@@ -327,13 +337,17 @@ public function backup_table( $search = '', $replace = '', $table, $new_table_pr
 						//skip replace if no search pattern
 						//check if we need to replace something
 						//skip primary_key
-						if ( $search !== '' && $column !== $primary_key ) {
+						if ( $search !== '' && $column !== $primary_key && $column !== "guid") {
 
 							$edited_data = $this->replace->recursive_unserialize_replace(
 								$search, $replace,
 								$value
 							);
-
+							if($csv != null) {
```

Comment:
We need whitespace, like in this example `if ( $csv !== null ) {` and also a identical usage, strict `!==` instead of `==`.